frontend/src/components/Buttons/TypeButton.vue

```diff
@@ -28,29 +28,35 @@
 </template>
 
 <script setup lang="ts">
+import { BaseItemKind } from '@jellyfin/sdk/lib/generated-client';
 import { computed, ref, mergeProps } from 'vue';
+import { useDisplay } from 'vuetify';
 import { useI18n } from 'vue-i18n';
 
-const { t } = useI18n();
 const props = withDefaults(
   defineProps<{
-    type: string;
+    /** CollectionType */
+    type: string | undefined;
     disabled?: boolean;
   }>(),
   { disabled: false }
 );
 const emit = defineEmits<{
-  (e: 'change', types: Record<string, string>): void;
+  (e: 'change', types: BaseItemKind | undefined): void;
```

Comment:
Thank you for replacing the remnants of not having ``BaseItemKind`` properly typed!